frontend/src/components/editor-page/sidebar/specific-sidebar-entries/revisions-sidebar-entry/revisions-modal/revision-modal-footer.tsx

```diff
@@ -48,15 +57,33 @@ export const RevisionModalFooter: React.FC<RevisionModalFooterProps & Pick<Modal
       .catch(showErrorNotification(''))
   }, [noteIdentifier, selectedRevisionId, showErrorNotification])
 
+  const openDeleteModal = useCallback(() => {
+    onHide?.()
+    onShowDeleteModal()
+  }, [onHide, onShowDeleteModal])
+
   return (
     <Modal.Footer>
       <Button variant='secondary' onClick={onHide}>
         <Trans i18nKey={'common.close'} />
       </Button>
-      <Button variant='danger' disabled={selectedRevisionId === undefined} onClick={onRevertToRevision}>
+      {showDeleteRevisionButton && (
+        <Button variant='danger' onClick={openDeleteModal} {...cypressId('revision.modal.delete.button')}>
+          <Trans i18nKey={'editor.modal.deleteRevision.button'} />
+        </Button>
+      )}
```

Comment:
You should show the button but disable it. Otherwise this could lead to confusion.

Reply:
sure